src/test/objectstore/test_bluefs.cc

```diff
@@ -1013,6 +1013,55 @@ TEST(BlueFS, test_update_ino1_delta_after_replay) {
   fs.umount();
 }
 
+TEST(BlueFS, truncate_fsync) {
```

Comment:
A general comment on the test -
* I think we should use stronger test running a loop with different size writes followed by truncate
* * I suggest using constants instead of magic-numbers like 1048576 or hard-coded names like "dir"/"file"/"foo"

Reply:
fixed